nucleus/cluster/cli/src/main/java/com/sun/enterprise/admin/cli/cluster/ChangeNodeMasterPasswordCommand.java

```diff
@@ -179,8 +181,7 @@ protected void createMasterPasswordFile() throws CommandException {
             PasswordAdapter p = new PasswordAdapter(pwdFile.getAbsolutePath(),
                 MASTER_PASSWORD_ALIAS.toCharArray());
             p.setPasswordForAlias(MASTER_PASSWORD_ALIAS, newPassword.getBytes());
-            pwdFile.setReadable(true);
```

Comment:
The side affect is that the file will not be explicitly set to be readable by non-owners, but will only have the default permission.  If the default permissions are such that only the owner can read, then these permissions will not be relaxed for the file.